tests/e2e-cypress/integration/topics.cy.js

```diff
@@ -270,4 +285,31 @@ describe('Topics', () => {
             })
         })
     })
+    context('Using the CatalogNodes URL parameter', () => {
+	it('Open the catalog and only the relevant topic and node elements when the URL contains the catalogNode parameter', () => {
+	    cy.goToMapView('en', {
+	    catalogNodes: '2'})
```

Comment:
the format is a bit weird here, are you sure you've run prettier on this file?

Reply:
the prettier command we made is not formatting the tests. I'll add this to another PR if that's okay with you.

Reply:
then fix your indentation (but you could also run prettier from your IDE on demand), the portion of code you're adding is not very well indented or has weird line breaks

for VS Code, there's a Prettier plugin, then you can use CTRL+SHIFT+P and boom, voilà, it's formatted